templates/workspaces/base/porter.yaml

```diff
@@ -157,6 +157,8 @@ upgrade:
       command: echo
       arguments:
         - "This workspace does not implement upgrade action"
+        - "Please be aware that parameter client_secret and auth_client_secret are redacted"
```

Comment:
Not sure auth_client_secret is redacted - that is a credential that is passed in by the RP

Reply:
Oh yes I forgot to fix that! Thanks